### PR TITLE
[ Scatter ] Adds new operation to opset3

### PR DESCRIPTION
**Pull request by jane-intel**

### CODE REVIEW

**Comment by silee2** [comment]
@jane-intel General CI fails with `ngraph/src/ngraph/op/util/scatter.cpp:26:12: error: unused variable 'AXIS' [-Werror,-Wunused-variable]`

File: src/ngraph/op/scatter_elements_update.cpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+#include "ngraph/op/scatter_elements_update.hpp"
+#include "ngraph/shape.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+constexpr NodeTypeInfo op::v3::ScatterElementsUpdate::type_info;
+
+shared_ptr<Node> op::v3::ScatterElementsUpdate::copy_with_new_args(const NodeVector& new_args) const
```
Comment (diyessi) [inline src/ngraph/op/scatter_elements_update.cpp]: Deprecated for some time. Switch it to `clone_with_new_inputs`.

File: src/ngraph/op/scatter_elements_update.hpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                                      const Output<Node>& updates,
+                                      const Output<Node>& axis);
+
+                void generate_adjoints(autodiff::Adjoints& /* adjoints */,
+                                       const OutputVector& /* deltas */) override
+                {
+                    throw ngraph_error("Not yet implemented");
+                }
+
+                virtual std::shared_ptr<Node>
```
Comment (diyessi) [inline src/ngraph/op/scatter_elements_update.hpp]: `copy_with_new_inputs`

File: src/ngraph/op/scatter_update.cpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+#include "ngraph/op/scatter_update.hpp"
+#include "ngraph/shape.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+constexpr NodeTypeInfo op::v3::ScatterUpdate::type_info;
+
+shared_ptr<Node> op::v3::ScatterUpdate::copy_with_new_args(const NodeVector& new_args) const
```
Comment (diyessi) [inline src/ngraph/op/scatter_update.cpp]: `copy_with_new_inputs`

File: src/ngraph/op/scatter_update.hpp [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                              const Output<Node>& updates,
+                              const Output<Node>& axis);
+
+                void generate_adjoints(autodiff::Adjoints& /* adjoints */,
+                                       const OutputVector& /* deltas */) override
+                {
+                    throw ngraph_error("Not yet implemented");
+                }
+
+                virtual std::shared_ptr<Node>
```
Comment (diyessi) [inline src/ngraph/op/scatter_update.hpp]: `copy_with_new_inputs`

**Comment by ilyachur** [comment]
Please add tests for new ops.

**Comment by arogowie-intel** [comment]
The work is moved to separate PR. It will come soon.

**Pull request closed without merging**